website/highlights/2020-07-10-add-reduce-transform.md

```diff
@@ -0,0 +1,52 @@
+---
+last_modified_on: "2020-07-15"
+$schema: "/.meta/.schemas/highlights.json"
+title: "New Reduce transform"
+description: "More flexibility around grouping events."
+author_github: "https://github.com/hoverbear"
+hide_on_release_notes: false
+pr_numbers: [2870]
+release: "0.10.0"
+tags: ["type: new feature","domain: transforms"]
+---
+
+You can now find a new [Reduce][urls.vector_transform_reduce]! This allows you to turn a stream of many small events into a stream of less small events!
+
+Say you had a stream of events like:
```

Comment:
Could we update this with a real-world example? Like reducing Rails request logs to a single transaction. You can see an example in the [original issue](https://github.com/timberio/vector/issues/1437).